lib/class-wp-rest-widget-types-controller.php

```diff
@@ -427,6 +468,106 @@ public function get_widget_form( $request ) {
 		);
 	}
 
+	/**
+	 * An RPC-style endpoint which can be used by clients to turn user input in
+	 * a widget admin form into an encoded instance object.
+	 *
+	 * Accepts:
+	 *
+	 * - id:        A widget type ID.
+	 * - instance:  A widget's encoded instance object. Optional.
+	 * - form_data: Form data from submitting a widget's admin form. Optional.
+	 *
+	 * Returns:
+	 * - instance: The encoded instance object after updating the widget with
+	 *             the given form data.
+	 * - form:     The widget's admin form after updating the widget with the
+	 *             given form data.
+	 *
+	 * @param WP_REST_Request $request Full details about the request.
+	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
+	 */
+	public function encode_form_data( $request ) {
+		$id            = $request['id'];
+		$widget_object = gutenberg_get_widget_object( $id );
+
+		$widget_object->_set( 1 );
```

Comment:
This line could use a comment, particularly since `_set` is "internal".

Reply:
👍 333fac4ae85f5316fd565a8ea48792c7204c423e